analysisapi/infrastructure/diary_repository_failed_exception.py

```diff
@@ -0,0 +1,17 @@
+"""
+データベースへ文章を追加することに失敗した場合の例外クラスです。
+"""
+
+from systemcommon.business_exception import BusinessException
+
+
+class DiaryRepositoryFailedException(BusinessException):
+    """
+    文章の分割に失敗した場合の例外クラスです。
+    Attributes:
+        user_id (int): ユーザーID 。
+    """
+
+    def __init__(self):
+        message = f"CHUNK_OVERLAP は CHUNK_SIZE 未満を指定してください"
```

Comment:
f文字列を使用するのであれば正しくは以下ではないでしょうか？
```suggestion
        message = f"{InfrastructureConstants.CHUNK_OVERLAP} は {InfrastructureConstants.CHUNK_SIZE} 未満を指定してください"
```
合わせて各定数のimport文を追加して下さい。